ui/src/literature/containers/CiteAllActionContainer.jsx

```diff
@@ -1,11 +1,16 @@
 import { connect } from 'react-redux';
-import omit from 'lodash.omit';
 
+import omit from 'lodash.omit';
 import CiteAllAction from '../components/CiteAllAction';
+import { searchScopes } from '../../reducers/search';
 
 const stateToProps = state => ({
   numberOfResults: state.search.get('total'),
-  query: omit(state.router.location.query, ['sort', 'page']),
+  query: {
+    // set default `sort` in case there is no `sort` in the location query
+    sort: searchScopes.getIn(['literature', 'query', 'sort']),
```

Comment:
it's a bit weird to not to use `state` in `stateToProps`, perhaps this could go inside the component itself. But i'd say optional since it's not a problem for `EmbeddedSearch` yet.

Reply:
we are using state as well, we are combining this with the state

Reply:
I meant that this particular logic having default `sort`, doesn't need the app state at all.